### ISSUE TITLE
WCAG 1.3.1 test 1: false positive

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#109]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7548220/test-document-3.pdf) fails **WCAG 1.3.1 test 1** ("_ASCII text is used as a pseudographics_"). I have no clue why this happens, the error seems to point to an ordinary text paragraph with no repeated characters.

Can you check what is going on here?

**Comment by bdoubrov** [comment]
The issue is cased by repeated spaces at the end of the paragraph. There are two thoughts here:

**Comment by bdoubrov** [comment]
Discussed and resolved as follows:
- introduce different checks for repeated spaces and non-spaces
- in case of repeated spaces add an additional error argument specifying where these repeated spaces occur (START, MIDDLE, END)

In addition, detect use of empty (or space only) paragraphs for vertical formatting (will be implemented as a separate feature and is not a part of this issue).

**Comment by jzuidweg** [comment]
WCAG 1.3.1 test 1 ("ASCII text is used as a pseudographics") has been re-implemented as different tests. This resolves this issue, which can be closed.
